scalableminds/webknossos
# Refactor Comments as Antd <Tree/>

We have three different tree hierarchy implementations in WK for skeletons, comments and segments. In this PR I refactor the comments tab to use the  `antd` `<Tree />` component, similar to the Segments tab.

I also updated the Comments Tab to be a React functional component.

### URL of deployed dev instance (used for testing):
- https://commenttree.webknossos.xyz/

### Steps to test:
- Create a few skeletons and create comments for them
  - Add a few markdown comments
- Create empty tree with no nodes (and hence no comments)
- Import a large NML with many comments and check performance

### TODOs:
- [x] Test rendering performance

### Issues:
- Contributes to #7765 

------
(Please delete unneeded items, merge only when none are left open)
